src/main/java/org/prebid/server/spring/config/bidder/model/BidderConfigurationProperties.java

```diff
@@ -38,5 +38,7 @@
     @NotNull
     private UsersyncConfigurationProperties usersync;
 
+    private List<BidderAccount> accounts;
```

Comment:
Please see how this is handled for FacebookBidder (`FacebookConfigurationProperties`). There is no need to add this field to generic class for properties.